src/persist/updatableAdapter.ts

```diff
@@ -16,7 +16,7 @@ import { Adapter } from './adapter';
 
 // UpdatableAdapter is the interface for Casbin adapters with update policy functions.
 export interface UpdatableAdapter extends Adapter {
-  // updatePolicies adds policy rules to the storage.
+  // updatePolicy updates a policy rules from the storage.
```

Comment:
`a policy rules` is wrong English, how is Golang code written here? Actually you can just copy from Go comments? Why don't do this?